content/rust/docker/rust-docker-arm.md

```diff
@@ -12,3 +12,365 @@ tags:
 - docker
 - arm
 ---
+
+This guide will walk you through the steps required to build and publish arm64 Docker images for Rust projects. This guide assumes you have a basic understanding of Rust and Docker, and that you have Rust and Docker installed.
+
+# Setup a Rust Project
+
+First we setup an example project. To do this we can use `cargo`:
+
+```
+$ cargo init hello-world
+```
+
+This will create a folder named `hello-world` which contains the project. It consists of a project file `Cargo.toml` and a source file `main.rs` in the `src` subdirectory. The source file contains code which simply prints `Hello, world!` in the console:
+
+{{< highlight text "linenos=table,hl_lines=99" >}}
+fn main() {
+    println!("Hello, world!");
+}
+{{< / highlight >}}
+
+Use the `cargo` tool to compile our program to a standalone binary file, and then execute the binary to start it:
```

Comment:
for 

```
the `cargo` tool
```

replace with just

```
`cargo`
```

again linking to the Cargo reference page would be helpful.